src/bundle/Resources/public/scss/_tabs.scss

```diff
@@ -1,24 +1,25 @@
 @import 'custom';
 
 .nav-tabs {
-    border-bottom: 2px solid $ez-ground-base-pale;
-    background-color: $ez-ground-base;
+    border-bottom: 2px solid $ez-ground-base-dark;
+    background-color: $ez-ground-base-dark;
 
     .nav-item {
         margin-bottom: 0;
     }
 
     .nav-link {
-        color: $ez-color-base-pale;
+        color: $ez-color-base-medium;
 
         &:hover {
-            background-color: $ez-ground-base-dark;
-            border-color: $ez-ground-base-dark;
+            background-color: $ez-color-base-pale;
+            border-color: $ez-color-base-pale;
```

Comment:
What's the point in setting the border color the same as the background color?
From what I can see in this PR: `.nav-tabs` and `.nav-link` items share the same colors for border and background. We might skip the border CSS properties, do you agree?

Reply:
Because when removed `background-color` and `border-color` are different.